test/e2e/mock-3box/server.js

```diff
@@ -0,0 +1,37 @@
+const http = require('http')
+const url = require('url')
+const port = 8889
+
+const database = {}
+
+const requestHandler = (request, response) => {
+  response.setHeader('Content-Type', 'application/json')
+  if (request.method === 'POST') {
+    let body = ''
+    request.on('data', chunk => {
+      body += chunk.toString() // convert Buffer to string
+      // console.log('Mock3Box POST body', body)
```

Comment:
Probably no need to keep this - I assume it was just used during testing.

Reply:
fixed in acd4f9bc7